app/scripts/controllers/HomeCtrl.js

```diff
@@ -2,6 +2,14 @@
   function HomeCtrl(Room, $uibModal) {
 
     this.roomObject = Room;
```

Comment:
Do you need this line?  The purpose is to allow your template access to the `Room` service.